bootstrap_sdk

```diff
@@ -83,6 +83,54 @@ mkdir -p "${ROOT_OVERLAY}/tmp"
 chmod 1777 "${ROOT_OVERLAY}/tmp"
 cp "${BUILD_LIBRARY_DIR}/toolchain_util.sh" "${ROOT_OVERLAY}/tmp"
 
+
+# Stage 1 uses a different ebuild repo snapshot as well as a different portage_overlay 
+# than the ones from the SDK where ./bootstrap_sdk is being executed -
+# to build a known-good stage 1 (see stages description at top
+# of file). This prevents package upgrades from updated ebuild files in the portage/coreos
+# to apply in stage 1.
+# Stage 1 lacks proper isolation and will link all packages built against its own
+# seed libraries instead of against libraries installed to /tmp/stage1root.
+build_stage1() {
+    local stage1_repos="$TEMPDIR/stage1-ebuild-repos"
+
+    write_configs
+
+    # use known-good gentoo base repo from SDK, coreos-overlay from manifest
+    info "Creating stage 1 ebuild repos and stage 1 snapshot in '$stage1_repos'"
+    rm -rf "$stage1_repos"
+    mkdir "$stage1_repos"
+    cp -R /var/gentoo/repos/gentoo "$stage1_repos"
+
+    local overlay_revision=$(
+        grep 'name="kinvolk/coreos-overlay"' /mnt/host/source/.repo/manifests/default.xml \
+        | sed 's/.*revision="refs\/\(heads\/\)\{0,1\}\([^"]\+\)".*/\2/' )
+
+    info "Using coreos-overlay revision '$overlay_revision'"
+
+    mkdir "$stage1_repos/coreos-overlay"
+    ( cd "$stage1_repos/coreos-overlay" \
+         && git clone https://github.com/kinvolk/coreos-overlay.git . \
```

Comment:
Why is this clone needed? Shouldn't it already be available in the build chroot?

Reply:
I agree it should, but it currently is not. I'm planning to address this in the near future - it's a super low-hanging fruit - but currently, coreos-overlay is not included in the chroot, only portage-stable is (in `/var/lib/gentoo/repos/gentoo`).

Reply:
Ok, it's not in /var/lib/gentoo/repos, but it is in `/mnt/host/source`, no?

Reply:
Not necessarily. Consider the following:
- User creates local SDK by means of `cork create` / `cork enter`
- User updates packages in `third_party/coreos-overlay`
- User rebuilds SDK
In the above case, updated packages would "leak" into the SDK build. It's not perfect - and it does not cover all cases - but it's the best we can do until we add a copy of coreos-overlay to the SDK tarball.

Reply:
I see, I'm not sure if this interferes with CI builds, I think from what you're saying that it doesn't, it's only for local user builds, but ok.